### PR TITLE
[FEAT] Adding Filtering on the Landing Page and Sorting to the Details Page 

### PR DESCRIPTION
**Pull request by twentymurial33**
"Closes https://github.com/twentymurial33/murials_menu/issues/83" User is now able to navigate to the filtered page in the Details Page for each food item 

- [ X] added filtering to the Landing Page so users can easily navigate to details page
- [X ]  added sorting to the details page 
Screen Shot 2023-04-04 at 7 52 41 PM

### CODE REVIEW

File: client/src/pages/Landing.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { createSearchParams, useNavigate } from "react-router-dom";
+// import { createSearchParams, useNavigate } from "react-router-dom";
```
Comment (brentmclark) [inline client/src/pages/Landing.js:1]: Can this be safely removed?

File: client/src/pages/Landing.js [hunk 2]
```diff
@@ -28,16 +28,16 @@ const images = [
...
-    });
-  };
+  // const openMenuItem = (q) => {
+  //   navigate({
+  //     pathname: `/Details`,
+  //     search: createSearchParams({
+  //       q: "lunch",
+  //     }).toString(),
+  //   });
+  // };
```
Comment (brentmclark) [inline client/src/pages/Landing.js]: Can this be safely removed?

File: client/src/pages/Landing.js [hunk 3]
```diff
@@ -59,7 +59,9 @@ export default function Landing() {
           >
             <ImageSrc style={{ backgroundImage: `url(${image.url})` }} />
             <Image>
-              <button onClick={openMenuItem}>Menu Item</button>
+              <button>
+                <a href={`/Details?q=${image.title}`}>Menu Item</a>
```
Comment (brentmclark) [inline client/src/pages/Landing.js:63]: :chefs_kiss:

File: client/src/pages/Landing.js [hunk 4]
```diff
@@ -1,46 +1,47 @@
...
+  // const navigate = useNavigate();
+
+  // const openMenuItem = (q) => {
+  //   navigate({
+  //     pathname: `/Details`,
+  //     search: createSearchParams({
+  //       q: "lunch",
+  //     }).toString(),
+  //   });
+  // };
```
Comment (brentmclark) [inline client/src/pages/Landing.js:40]: Can this be safely removed?

**Review by brentmclark — COMMENTED** [review]
Looks great!  Is there a separate PR coming for the styles or is this one staying open?

File: client/src/pages/Landing.js (hunk 4 above)
Comment (twentymurial33) [reply client/src/pages/Landing.js:40]: Yes removed

File: client/src/pages/Landing.js (hunk 1 above)
Comment (twentymurial33) [reply client/src/pages/Landing.js:1]: Done

**Comment by twentymurial33** [comment]
> Looks great! Is there a separate PR coming for the styles or is this one staying open?

Yes , I will leave it open for CSS

**Comment by twentymurial33** [comment]
> > Looks great! Is there a separate PR coming for the styles or is this one staying open?
> 
> Yes , I will leave it open for CSS

Updated Button 
Screen Shot 2023-04-05 at 7 43 23 PM
CSS

**Review by brentmclark — APPROVED** [review]
Looks great!

**Pull request merged**